stream/distributedlog/core/pom.xml

```diff
@@ -146,7 +146,7 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-checkstyle-plugin</artifactId>
         <configuration>
-          <suppressionsLocation>distributedlog/suppressions.xml</suppressionsLocation>
+          <suppressionsLocation>buildtools/src/main/resources/distributedlog/suppressions.xml</suppressionsLocation>
```

Comment:
Is this path relative?
How can this work?
Does it work if you run
mvn validate
From a non root directory?

Reply:
It works. No idea why, but it does work.